modules/media/jsx/editForm.js

```diff
@@ -198,13 +198,6 @@ class MediaEditForm extends Component {
     e.preventDefault();
 
     let xhr = new XMLHttpRequest();
-    xhr.upload.addEventListener('progress', (evt) => {
```

Comment:
I don't believe the following removed code did anything visual for the user. I had the same experience when testing the media upload of a file.